google/validation.go

```diff
@@ -18,6 +19,7 @@ const (
 	SubnetworkLinkRegex = "projects/(" + ProjectRegex + ")/regions/(" + RegionRegex + ")/subnetworks/(" + SubnetworkRegex + ")$"
 
 	RFC1035NameTemplate = "[a-z](?:[-a-z0-9]{%d,%d}[a-z0-9])"
+	CloudIoTIdRegex     = "[a-zA-Z][-a-zA-Z0-9._+~%]{2,254}"
```

Comment:
Is this well-tested?  The formatter is trying to complain about the % as if it's a format-string specifier and I'm pretty sure it's a formatter bug and not a bug in your code.  :)

Reply:
Unit test is now included. Found bug and yep this seems to be a formatter bug :-)